doc/sphinx/language/core/modules.rst

```diff
@@ -645,7 +645,7 @@ while noting a few exceptional commands for which :attr:`local` and
       :cmd:`Definition`, :cmd:`Theorem` or their variants, its
       semantics are different: it makes the defined objects available
       only through their fully qualified names rather than their
-      unqualified names after an :cmd:`Import`.
+      unqualified names, even after an :cmd:`Import`.
```

Comment:
"rather than their unqualified names" seems like overkill if the previous clause says "_**only**_ through their unqualified names"